lib/features/home_page/widgets/pokemon_tile.dart

```diff
@@ -30,7 +30,7 @@ class _PokemonTileState extends State<PokemonTile> {
     Future.delayed(
       Duration.zero,
       () async {
```

Comment:
create private method and call it here, no need for Future.delayed. 

`void _getTypes() async {...}`